### PR TITLE
Feature/meeting and board handling data

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: src/03 - Code lists.md [hunk 1]
```diff
@@ -61,6 +61,43 @@ No default values are present in this list and must be configured in the system
 | S    | Saksbehandler har registrert innkommet dokument |         |
 | R    | Reservert dokument                              | &nbsp;  |
 
+### moeteregistreringstype (M085)
+
+| Code | Name                                                                        | Comment |
+|:-----|:----------------------------------------------------------------------------|:--------|
+| J    | Journalført                                                                 |         |
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/03 - Code lists.md]: Fixed.

File: src/03 - Code lists.md [hunk 2]
```diff
@@ -61,6 +61,43 @@ No default values are present in this list and must be configured in the system
...
+| U    | Uregistrert sak  |         |
+| D    | Delegert møtesak |         |
+| I    | Interpellasjon   | &nbsp;  |
+
+
+### moeteregistreringsstatus (M055)
+
+| Code | Name | Comment |
+|:-----|:-----|:--------|
+|      |      | &nbsp   |
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/03 - Code lists.md]: Fixed.

File: src/02 - Object types.md [hunk 3]
```diff
@@ -225,6 +225,42 @@ The following notation is used in the tables below:
...
+|      |                 |                               |          |             |           |  |      |      |       |      |               |                                                                                                      |           |
+| M711 | vendor-specific | virksomhetsspesifikkeMetadata |          |             |           |  |  x   |  x   |       |      |               |                                                                                                      |           |
+|      |                 |                               |          |             |           |  |      |      |       |      |               |                                                                                                      |           |
+|      | ref             | refArkivdel                   |    c     |             |           |  |      |  x   |   x   |      |       x       |                                                                                                      |           |
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Good point! Fixed.

File: src/02 - Object types.md [hunk 4]
```diff
@@ -289,6 +340,47 @@ The following notation is used in the tables below:
...
+| M085 | string          | moeteregistreringstype        |    x     |             |     x     |  |  x   |  x   |   x   |  x   |               | <p>&bull; code list in Documaster</p>                                                                |                                                                                                                                                         |
+| M088 | string          | moetesakstype                 |    x     |             |     x     |  |  x   |  x   |   x   |  x   |               | <p>&bull; code list in Documaster</p>                                                                |                                                                                                                                                         |
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Fixed.

File: src/01 - Archive models.md [hunk 5]
```diff
@@ -31,3 +31,10 @@ By definition, Noark 5 systems must support at least one of these models and may
 - Classification is mandatory
 
 ![Archive model 3](img/archive-model-three.png)
+
+## Archive model 4
+
+- Suitable for archiving meeting and board handling data in meeting folders along with other documents
+- Classification is optional but strongly mandatory
```
Comment (teodoradobreva) [inline src/01 - Archive models.md]: Reword to "Classification is optional but strongly **recommended**"?

File: src/03 - Code lists.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/03 - Code lists.md]: Fixed.

File: src/01 - Archive models.md (hunk 5 above)
Comment (ivaylomitrev) [reply src/01 - Archive models.md]: Fixed.

File: src/02 - Object types.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Fixed as part of a previous comment by @teodoradobreva

File: src/02 - Object types.md [hunk 6]
```diff
@@ -225,6 +225,42 @@ The following notation is used in the tables below:
...
+|      | ref             | refPrimaerKlasse              |          |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references a **Klasse** from the primary **Klassifikasjonssystem** of the **Arkivdel**</p> |           |
+|      | ref             | refSekundaerKlasse            |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; references a **Klasse** from any **Klassifikasjonssystem**</p>                             |           |
+|      | ref             | refEksternId                  |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; *unlink* not supported in Documaster</p>                                                   | &nbsp;    |
+|      | ref             | refNoekkelord                 |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; *unlink* not supported in Documaster</p>                                                   | &nbsp;    |
+
 ### Sakspart
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Re-arranged the document to have the following sequence of objects:

- Case File
- Case Party
- Meeting Folder
- Meeting Participant

File: src/02 - Object types.md [hunk 7]
```diff
@@ -251,6 +287,21 @@ The following notation is used in the tables below:
 |      |        |                        |          |             |           |  |      |      |       |      |               |                                                                                          |           |
 |      | ref    | refMappe               |    x     |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references **Saksmappe**</p><p>&bull; *unlink* not supported in Documaster</p> | &nbsp;    |
 
+### Moetedeltaker
```
[the remark this answers is not in the document]
Comment (ivaylomitrev) [reply src/02 - Object types.md]: Re-arranged the document to have the following sequence of objects:

- Case File
- Case Party
- Meeting Folder
- Meeting Participant

**Comment by ivaylomitrev** [comment]
Squashed in preparation for merge. No changes introduced.

**Pull request merged**